contracts/triggers/AaveV2RatioTrigger.sol

```diff
@@ -5,10 +5,13 @@ pragma solidity =0.8.10;
 import "../auth/AdminAuth.sol";
 import "../actions/aave/helpers/AaveRatioHelper.sol";
 import "../interfaces/ITrigger.sol";
+import "../utils/TransientStorage.sol";
 
 contract AaveV2RatioTrigger is ITrigger, AdminAuth, AaveRatioHelper {
 
     enum RatioState { OVER, UNDER }
+
+    TransientStorage public constant tempStorage = TransientStorage(TRANSIENT_STORAGE);
```

Comment:
We should probably think about adding transient storage to registry and fetching transient storage from there instead of hardcoding everywhere. Thoughts on this?

Reply:
Transient mainly used in few specific triggers and checkers so would keep it as is, for now